### ISSUE TITLE
nodeset: --autostep default value

### ISSUE THREAD

**Issue by thiell** [cea-hpc/clustershell#161]
We could provide a default autostep value when only `--autostep` option is specified. A first idea is to set this value to the total number of nodes, so that only global autostep factorization will take place, so for example:

```bash
$ nodeset --autostep -f node6 node2 node4 node8
node[2-8/2]

$ nodeset --autostep -f node6 node2 node4 node8 node11
node[2,4,6,8,11]
```

Another solution is to set a default autostep value in relation to the number of nodes n, for example n/3, or also to set a fixed autostep default value (eg. 5) ?

**Comment by thiell** [comment]
I have a draft ready for this feature (local implementation for CLI/Nodeset.py only), but I need a fix for #256 first to avoid string cast.

**Comment by thiell** [comment]
https://review.gerrithub.io/238955
